CodeEdit/Features/Settings/Pages/SourceControlSettings/SourceControlGitView.swift

```diff
@@ -11,36 +11,119 @@ struct SourceControlGitView: View {
     @AppSettings(\.sourceControl.git)
     var git
 
-    @State var ignoredFileSelection: IgnoredFiles.ID?
+    let gitConfig = GitConfigClient(shellClient: currentWorld.shellClient)
+
+    @State private var authorName: String = ""
+    @State private var authorEmail: String = ""
+    @State private var defaultBranch: String = ""
+    @State private var preferRebaseWhenPulling: Bool = false
+    @State private var hasAppeared: Bool = false
+    @State private var resolvedGitIgnorePath: String = "~/.gitignore_global"
 
     var body: some View {
-        SettingsForm {
+        Group {
             Section {
                 gitAuthorName
                 gitEmail
+            } header: {
+                Text("Git Configuration")
+                Text("""
+                Applied globally to all repositories on your Mac. \
+                [Learn more...](https://git-scm.com/docs/git-config)
+                """)
             }
             Section {
+                defaultBranchName
                 preferToRebaseWhenPulling
                 showMergeCommitsInPerFileLog
             }
+            Section {
+                gitConfigEditor
+            }
+            Section {
+                IgnoredFilesListView()
+            } header: {
+                Text("Ignored Files")
+                Text("""
+                Patterns for files and folders that Git should ignore and not track. \
+                Applied globally to all repositories on your Mac. \
+                [Learn more...](https://git-scm.com/docs/gitignore)
+                """)
+            }
+            Section {
+                gitIgnoreEditor
+            }
+        }
+        .onAppear {
```

Comment:
Can we get a comment on this describing why it's not a `.task`? I know a lot of experimentation went into the decision